src/arrayloaders/abc.py

```diff
@@ -90,12 +110,23 @@ def __init__(
             raise NotImplementedError(
                 "If you need batch loading that is bigger than the iterated in-memory size, please open an issue."
             )
+
+        for package, arg, arg_name in [
+            ("torch", to_torch, f"{to_torch=}"),
+            ("cupy", preload_to_gpu, f"{preload_to_gpu=}"),
+        ]:
+            if arg and not find_spec(package):
+                raise ImportError(
+                    f"Could not find {package} dependency even though {arg_name}.  Try `uv pip install {package}`"
```

Comment:
```suggestion
                    f"Could not find {package} dependency even though {arg_name}.  Try `pip install {package}`"
```

Nit: Everyone has pip but not everyone has uv. The people that uv though will know how to install things with uv.